src/framework/parsers/texture/img.js

```diff
@@ -20,6 +23,13 @@ class ImgParser {
         this.crossOrigin = registry.prefix ? 'anonymous' : null;
         this.maxRetries = 0;
         this.device = device;
+
+        // run image alpha test
+        Debug.call(() => {
```

Comment:
why do we run this in debug mode only?

Reply:
try reading the PR description ;)

Reply:
Ah I see.
Perhaps add `// #if _DEBUG` around `import { ImgAlphaTest }` to exclude the code from non-debug builds.

Reply:
The code is already excluded from non-debug builds because it's not referenced anywhere (and not exported). Might be worth adding the #if anyway to make that more obvious to readers. Thanks!